packages/terra-button-group/src/ButtonGroup.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+import ButtonGroupButton from './ButtonGroupButton';
+import './ButtonGroup.scss';
+
+const propTypes = {
+  /**
```

Comment:
This seems pretty picky to me. I'm all for keeping code uniform but my vote is keeping that uniformity to be the responsibility of the linter. If you guys feel strongly about ordering props, update the linter to enforce that prop order instead of documenting the standard wiki. That way anyone who checks in will have the correct format and we won't have to look for it in code reviews.